src/config/cron.md

```diff
@@ -1,13 +1,28 @@
 # Cron
 
-Void Linux comes without a default cron daemon, you can choose one of multiple
-cron implementations, by installing the package and enabling the system service.
+[cron](https://en.wikipedia.org/wiki/Cron) is a daemon for running programs at
+regular intervals. The programs and intervals are specified by using
+[crontab(1)](https://man.voidlinux.org/crontab.1) to edit a `crontab` file.
+`crontab` files can be system-wide or user-specific; to edit the system crontab,
+run `crontab` as the superuser:
 
-Available choices include [cronie](https://github.com/cronie-crond/cronie/),
+```
+# crontab -e
+```
+
+Similarly, run `crontab -e` under your own login to edit your personal crontab.
+
+By default, a cron daemon is not installed. However, multiple cron
+implementations are available, including
+[cronie](https://github.com/cronie-crond/cronie/),
 [dcron](http://www.jimpryor.net/linux/dcron.html),
 [fcron](http://fcron.free.fr/) and more.
 
-As alternative to the standard cron implementations you can use something like
-[snooze](https://github.com/leahneukirchen/snooze) or
-[runwhen](https://code.dogmap.org/runwhen/) which go hand in hand with service
-supervision.
+Once you have chosen and installed an implementation,
+[enable](./services/index.md#enabling-services) the `crond` service.
+Implementation documentation will be available in
+[crond(8)](https://man.voidlinux.org/crond.8).
```

Comment:
Perhaps make sure to mention that if you have several installed it is necessary to use `xbps-alternatives` to choose between them, or at least that the `crond` service is controlled by alternatives.

Reply:
Sure, will do.